apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/RegexUtils.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.google.common.collect.ImmutableList;
+
+/** Helper Utils class for Functions shared across. multiple classes. */
+public class RegexUtils {
+
+  /**
+   * Custom Split Function to replace.
+   *
+   * @param input a String that needs to remove whitespace and commas.
+   * @return Split Array.
+   */
+  public static String[] customSplit(String input) {
+    final ImmutableList.Builder<String> builder = ImmutableList.builder();
+    StringBuilder curr = new StringBuilder();
```

Comment:
Plus 1000 for using `ImmutableList` here!

Reply:
Thanks